packages/calcite-components/src/components/loader/loader.e2e.ts

```diff
@@ -55,4 +55,43 @@ describe("calcite-loader", () => {
     expect(loader).toHaveAttribute("id");
     expect(loader.getAttribute("id").length).toEqual(36);
   });
+
+  it("animates loader segment offset", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-loader></calcite-loader>`);
+
+    const duration = await page.evaluate(() => {
+      const loader = document.querySelector("calcite-loader");
+      const style = getComputedStyle(loader);
```

Comment:
I think you should be able to do: `const style = getComputedStyle(loader).animationDuration;`

Reply:
Applies to other places.